docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -2,4 +2,191 @@
+builds a ``uwtools`` package, so it is important to periodically run
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+
+The order of the targets above is intentional, and possibly useful:
+
+   * ``make format`` will complain about certain kinds of syntax errors that
+     would cause all the remaining code-quality tools to fail (and may change
+     line numbers reported by other tools, if it ran after them).
+   * ``make lint`` provides a good first check for obvious errors and
+     anti-patterns in the code.
+   * ``make typecheck`` offers a more nuanced look at interfaces between 
+     functions, methods, etc. and may spot issues that would cause 
+     ``make unittest`` to fail.
+
+
+In addition to the ``make devshell`` command, other ``make`` targets
+are available for use *outside* a development shell, i.e. from the base
+conda environment (requires presence of the ``condev`` package):
+
+
++------------------+-------------------------------------------------------+
+| Command          | Description                                           |
++==================+=======================================================+
+| ``make env``     | Creates a conda environment based on the ``uwtools``  |
+|                  | code                                                  |
++------------------+-------------------------------------------------------+
+| ``make meta``    | Update ``recipe/meta.json`` from ``recipe/meta.yaml`` |
++------------------+-------------------------------------------------------+
+| ``make package`` | Builds a ``uwtools`` conda package                    |
++------------------+-------------------------------------------------------+
+
+
+These targets work from the code in its current state in the clone.
+``make env`` calls ``make package`` automatically to create a local
+package, then builds an environment based on the package.
+
+Building condev locally
+-----------------------
+
+As an alternative to installing a prebuilt package from
+`anaconda`_, the ``condev`` package can be
+built locally, then installed into the local conda installation. 
+Ensure that ``conda-build`` and ``conda-verify`` are installed in the base
+environment:
+
+   .. code:: sh
+
+     # Activate your conda
+     git clone https://github.com/maddenp/condev.git
+     make -C condev package
+     conda install -y -c local condev
+
+Files derived from condev
+-------------------------
+
+The following files in this repo are derived from their counterparts in
+the `condev demo`_:
+
+
+   .. code:: sh
+
+     ├── Makefile
+     ├── recipe
+     │   ├── build.sh
+     │   ├── channels
+     │   ├── meta.json
+     │   ├── meta.yaml
+     │   └── run_test.sh
+     ├── src
+     │   ├── pyproject.toml
+     │   ├── setup.py
+
+Behaviors described in previous sections may rely on these files
+continuing to follow ``condev`` conventions.
```

Comment:
I'm not sure about the relevance of these lines here. I debated on moving them to another section, but as I read more I'm left wondering what does the "in previous sections" refer to?  

There's a lot of content for the Developers Setup, is it talking about the documentation as a whole? I think more context is needed to give better understanding.

Reply:
Yes, that definitely clears the confusion up. The last line didn't feel like it fit anywhere in particular, but paired with the line before the code block helps a lot.